web/src/main/java/se/inera/intyg/minaintyg/monitoring/MonitoringLogServiceImpl.java

```diff
@@ -34,7 +39,8 @@ private String buildMessage(MonitoringEvent logEvent) {
 
   private enum MonitoringEvent {
     CITIZEN_LOGIN("Citizen '{}' logged in using login method '{}'"),
-    CITIZEN_LOGOUT("Citizen '{}' logged out using login method '{}'");
+    CITIZEN_LOGOUT("Citizen '{}' logged out using login method '{}'"),
+    LIST_CERTIFICATES("'{}' certificates listed for patient");
```

Comment:
Consider changing the log message. Good to log both number of certificates and which citizen. Maybe follow similar pattern as the others. 

LIST_CERTIFICATES("Citizen '{}' listed  '{}' certificates")